aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -1679,7 +1677,7 @@ static Tensor& _linalg_norm_vector_out(Tensor& result, const Tensor& self, optio
       // when the input contains extreme values (like nan or +/-inf) or if the input
       // size is degenerate (like size(0), size(0, N), etc)
       case_was_overridden = true;
-      self_ = self.abs();
+      self_ = self_.abs();
```

Comment:
why don't we just declare self_ without assignment above?

Reply:
The conditional dtype conversion `opt_dtype.has_value() ? self.to(opt_dtype.value()) : self` needs to be performed for all cases, which is the reason why `self_` is defined